rust/Cargo.lock.in

```diff
@@ -3,10 +3,10 @@
 version = 3
 
 [[package]]
-name = "adler"
-version = "1.0.2"
+name = "adler2"
+version = "2.0.0"
```

Comment:
@jasonish are we forced to update everything, before updating a direct dependency ?

Reply:
You can get more precise if needed.

Reply:
For my information, what is the good way to proceed to update only one crate ?

Reply:
> For my information, what is the good way to proceed to update only one crate ?

Never really done that.  If not updating `Cargo.toml` you can do `cargo update <package>` just to update a single package in the lock file.

If updating `Cargo.toml`, its probably a little more difficult as you'd have to figure out what transitive updates are needed or not.  So update `Cargo.toml`, build, which will regenerate `Cargo.lock`, and figure out whats needed.